### DISCUSSION TITLE
外部控制 Dashboard 版本无法使用meta版本并更新回官方版本失败

### DISCUSSION THREAD

**Discussion by Ronan0001** [vernesong/OpenClash#3678] [Q&A] [answered]
我习惯使用 Dashboard版本来查看节点信息，
但是会因为一些其他的不知名的操作（比如切换节点）又变回官方版本
![{F8CDDBDD-D63B-4762-AB54-D3E4B960CE5E}](https://github.com/vernesong/OpenClash/assets/150348920/b4e69a3b-bbf0-437e-94f5-2fcbef5df1cc)
显示的还是在meta版本并且更新回官方会显示失败
这种应该怎么处理

**Comment by ghost** [comment]
https://github.com/vernesong/OpenClash/issues/3597

**Reply by Ronan0001** [reply]
我是想用meta版本  直接先替换回官方再直接更新么？

**Reply by ghost** [reply, marked as answer]
这种情况啊，卡死了，只能手动上传你需要的dashboard版本

但是

它应该不会乱换的，应该是你没清浏览器cookies和缓存
用隐私模式开试试会不会乱换


姑且也把meta dashboard的下载地址给你吧
https://codeload.github.com/MetaCubeX/Razord-meta/zip/refs/heads/gh-pages
